JS/cart.js

```diff
@@ -147,99 +162,113 @@ function createCart2(article) {
      * We update the SessionStorage by deleting an entry .
      * We also update the component that show the number of article in the cart.
     */
-    del_text_div.onclick = function () {
+    del_text_div.onclick = function () {/*
```

Comment:
is this huge block of commented code useful ?

Reply:
It's no longer useful now that I've created a pop up.